cirq/experiments/fidelity_estimation_test.py

```diff
@@ -0,0 +1,59 @@
+from typing import Sequence
+
+import numpy as np
+import pytest
+
+import cirq
+
+
+def make_bitstrings(samples: np.ndarray) -> Sequence[int]:
+    assert samples.shape[1] == 2
+    return [2 * b1 + b0 for b1, b0 in samples]
+
+
+def sample_noisy_bitstrings(circuit: cirq.Circuit, depolarization: float,
+                            n_samples: int) -> Sequence[int]:
+    assert 0 <= depolarization <= 1
+    n_incoherent = int(depolarization * n_samples)
+    n_coherent = n_samples - n_incoherent
+    incoherent_samples = np.random.randint(2, size=(n_incoherent, 2))
+    if n_coherent > 0:
+        sim = cirq.Simulator()
+        r = sim.run(circuit, repetitions=n_coherent)
+        coherent_samples = r.measurements['0,1']
```

Comment:
I think `r.data['0,1']` has the ints that you want.

Reply:
Cool, thanks!